node/countries/src/server.js

```diff
@@ -0,0 +1,50 @@
+const _ = require("lodash");
+const express = require("express");
+const app = express();
+
+const descriptor = require("./descriptor");
+const de = require("cldr-localenames-modern/main/de/territories");
+const en = require("cldr-localenames-modern/main/en/territories");
+const es = require("cldr-localenames-modern/main/es/territories");
+const fr = require("cldr-localenames-modern/main/fr/territories");
+const data = _.merge({}, de, en, es, fr);
+
+// Serves the JSON type descriptor.
+app.get("/", function(req, res) {
+    res.json(descriptor);
+});
+
+// Serves CLDR countries, with ISO 3166-1 alpha-2 country codes and country names in the given language.
+app.get("/:language/options", function(request, response) {
```

Comment:
Isn't that connector missing the endpoint for `/:language/options/:id`?

Reply:
No, I don't think so - the type descriptor specifies `"fetchOneAvailable" : false`